tests/lib/rules/valid-template-root.js

```diff
@@ -119,6 +123,11 @@ tester.run('valid-template-root', rule, {
       errors: [
         "The template root with 'src' attribute is required to be empty."
       ]
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div v-if="foo"></div></template>',
+      errors: ['The v-if should not be used on root element without v-else.']
     }
```

Comment:
I think this new behavior should be configurable, since it is actually a valid template root (which this rule checks) and some users might want to allow this.

Reply:
Thank you, the option ignoreOnlyIfDirective has been added